resources/knative-eventing-kafka/templates/broker-secret.yaml

```diff
@@ -2,7 +2,8 @@
 {{- if hasKey .Values.kafka "brokers" }}
 apiVersion: v1
 data:
-  brokers: {{ .Values.kafka.brokers | b64enc }}
+{{/*  brokers: {{ .Values.kafka.brokers | b64enc }}*/}}
```

Comment:
this secret is azure specific, why are we templating it?

Reply:
to concatenate the hostname with the port